packages/api/src/router/role.ts

```diff
@@ -46,4 +46,66 @@ export const roleRouter = {
   delete: protectedProcedure.input(z.string()).mutation(({ ctx, input }) => {
     return ctx.db.delete(Role).where(eq(Role.id, input));
   }),
+
+  getAverageById: publicProcedure
+    .input(z.object({ roleId: z.string() }))
+    .query(async ({ ctx, input }) => {
+      const reviews = ctx.db.query.Review.findMany({
```

Comment:
I think you can put an `await` here so that you don't have to await reviews each time you use it below